dist/index.html

```diff
@@ -4,10 +4,33 @@
     <meta charset="utf-8" />
     <meta http-equiv="X-UA-Compatible" content="IE=edge">
     <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <link rel="stylesheet" href="https://cdnjs.cloudflare.com/ajax/libs/font-awesome/6.1.1/css/all.min.css" integrity="sha512-KfkfwYDsLkIlwQp6LFnl8zNdLGxu9YAA1QvwINks4PhcElQSvqcyVLLD9aMhXd13uQjoXtEKNosOWaZqXgel0g=="
+    crossorigin="anonymous"
+    referrerpolicy="no-referrer">
     <title>Getting Started</title>
     <script src="https://unpkg.com/lodash@4.17.20"></script>
   <script defer src="main.js"></script></head>
   <body>
+    <main class="main">
+        <div class="container">
+            <div class="list-head">
+            <h1 class="container-h">Today's to-do</h1>
+            <i class="fa-solid fa-arrows-rotate"></i>
+            </div>
+            <hr>
+            <form id="add-task-form">
+                <input type="text" placeholder="Add to your list..." class="add-task">
+                <!-- <input type="submit"><i class="fa-solid fa-arrow-turn-down-left"></i></input> -->
+            </form>
+            <hr>
+                <ul class="to-do"></ul>
+              <hr>
+              <div class="clear">
+                <span class="btn">Clear all completed</span>
+              </div>
```

Comment:
To make your code more semantic, it would be better to use section tags, in order to group your page major sections. (Please implement changes regarding emantic tags in your src/index.html file